### PR TITLE
test: set test path glob for SonarCloud analysis

### PR DESCRIPTION
**Pull request by alexandra-m-yarosh**
Hello here,

This change excludes `*spec.ts` files from SonarCloud static code and unit test coverage analysis.

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -2,4 +2,5 @@ sonar.projectKey=Frontify_app-bridge
 sonar.organization=frontify
 
 sonar.javascript.lcov.reportPaths=./coverage/lcov.info
-sonar.test.exclusions=./src/tests
+sonar.test=./src/tests, **/*spec.ts
```
Comment (SamuelAlev) [inline sonar-project.properties:5]:
```suggestion
sonar.test=./src/tests, **/*.spec.ts
```

File: sonar-project.properties [hunk 2]
```diff
@@ -2,4 +2,5 @@ sonar.projectKey=Frontify_app-bridge
...
+
```
Comment (SamuelAlev) [inline sonar-project.properties:6]: Extra line
```suggestion
```

**Comment by SamuelAlev** [comment]
@alexandra-m-yarosh Now we have `No Coverage information` on the SC report

**Comment by alexandra-m-yarosh** [comment]
> @alexandra-m-yarosh since the change to `sonar.test`, we have `No Coverage information` on the SC report

I think it's how the SC works for PRs. It analyses only changed code in PR under analysis.
If it contains code qualified for analysis, then it applies static analysis rules and code coverage from the reports we provide.

For example, I added code mutation in [f212c5c](https://github.com/Frontify/app-bridge/pull/191/commits/f212c5c19e757d57c453852c65d6df4fabc5e2bb), and it detected that 92% of the changed code was duplicated and had one code smell

Screenshot 2022-08-23 at 09 54 43

**Comment by triggertoo** [comment]
I wonder if you shouldn't just [exclude the files from code duplication](https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/), like:
```
sonar.cpd.exclusions=...
```

**Comment by alexandra-m-yarosh** [comment]
> I wonder if you shouldn't just [exclude the files from code duplication](https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/), like:
> 
> ```
> sonar.cpd.exclusions=...
> ```

Thanks for the suggestion  @triggertoo! I've run a test, and looks like that's what we need:

**Pull request closed without merging**